evidence/reactor.go

```diff
@@ -17,7 +17,7 @@ const (
 
 	maxMsgSize = 1048576 // 1MB TODO make it configurable
 
-	broadcastEvidenceIntervalS = 60  // broadcast uncommitted evidence this often
+	broadcastEvidenceIntervalS = 1   // broadcast uncommitted evidence this often
```

Comment:
I'm guessing this was just used for testing, and shouldn't remain like this?

Reply:
I'm not sure. I think 60 seconds is too high. This is basically the time we wait before we go back to the start of the list again and send the same evidence again. I think it probably should be a little more than the time it takes to make a block. So perhaps 5 - 10 seconds is better.

Reply:
Yeah, I wouldn't really know, but 1 second seems too aggressive at least.